### PR TITLE
Landing-Page Done

### PR DESCRIPTION
**Pull request by dharmyx**
- Blueprint of Footer done with HTML
- Styling with Css to look good

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 
+.vscode/settings.json
```
Comment (IbnMaruf) [inline .gitignore:2]: I suggest you run the git status command before adding your files

**Comment by dharmyx** [comment]
Good morning,
Please I would like to ask, was my pull request declined? 
Best regards,

Damilola Babalola+2347034707018teminidami@yahoo.co.uk 

    On Tuesday, 9 June 2020, 03:37:16 GMT+1, Abdullah Olamide Owolabi <notifications@github.com> wrote:  
 
 


@IbnMaruf commented on this pull request.

In .gitignore:
> @@ -1 +1,2 @@
 
+.vscode/settings.json

I suggest you run the git status command before adding your files

**Comment by dharmyx** [comment]
Well, maybe it's a new instruction, as at when I was doing this I followed the instructions as stated in the readme file.
Best regards,

Damilola Babalola+2347034707018teminidami@yahoo.co.uk 

    On Tuesday, 9 June 2020, 13:25:54 GMT+1, Sadiq A Sambo <notifications@github.com> wrote:  
 
 


@saslamp requested changes on this pull request.

You didn't follow the Instructions to pull from the upstream and get the updated branch.

**Comment by saslamp** [comment]
I mentioned that early this morning. Merging your code will cause conflict with the new structure that was done last night. Pull from the upstream & make any necessary modifications & push your branch again. It will be reviewed

**Pull request closed without merging**